### ISSUE TITLE
Ways to authenticate events?

### ISSUE THREAD

**Issue by mgmoens** [CenEkaGent/ceneka.ugent.be#32]
Discussion page. I'm not particularly opinionated at this moment in time and space.

**Comment by aavdiere** [comment]
How do you mean? The procedure to subscribe to an event or what should be visible on the site etc.? I would go for as similar to VTK as possible, they have a nice way of doing it.

**Comment by tverum** [comment]
Yeah that was discussed before, this should be done. Other examples include: how to limit the number of registrations for a certain event, how to do registrations in different timeslots etc. This are all features to be added at some point somewhere in the future.
